docs/components/App.js

```diff
@@ -55,7 +57,11 @@ export default function App({ children }: Props): Node {
           <ColorSchemeProvider colorScheme={colorScheme} id="gestalt-docs">
             <OnLinkNavigationProvider onNavigation={useOnNavigation}>
               <NavigationContextProvider>
-                <AppLayout>{children}</AppLayout>
+                <DocsExperimentProvider>
```

Comment:
We have to establish these providers for the docs. The experiment provider is a way to pass experiment names while testing out a feature — a poor man's version of ModHeader for now, until we come up with a better solution. The i18n provider is necessary as we throw errors if the expected strings aren't provided.

Reply:
(This now refers to the providers added in `_app.js`